pallets/handles/src/lib.rs

```diff
@@ -17,14 +17,13 @@
 //! * Homoglyph detection
 //!
 //! ## Terminology
-//! - Handle
-//! - Suffix
 //!
-//! ## Glossary
-//! - **Handle:** A handle is a unique identifier for a user. Handle on frequency is composed of a `base_handle`, its canonical version as, `canonical_handle` and a unique numeric `suffix`.
+//! Handle on frequency is composed of a `base_handle`, its canonical version as, `canonical_handle` and a unique numeric `suffix`.
+//!
 //! - **Base Handle:** A base handle is a user's chosen handle name.  It is not guaranteed to be unique.
+//! - **Display Handle:** A handle is a unique identifier for a user. Display handle is `base_handle`.`suffix`.
 //! - **Canonical Handle:** A canonical handle is a base handle that has been converted to a canonical form. Canonicals are unique representations of a base handle.
-//! - **Delimiter:** Period character (".") is reserved on Frequency to form full handle as `canonical_handle`.`suffix`.
+//! - **Delimiter:** Period character (".") is reserved on Frequency to form display handle as `canonical_handle`.`suffix`.
```

Comment:
I think this is referring to base handle right? The canonical handle never exists in a string with the suffix I think

```suggestion
//! - **Delimiter:** Period character (".") is reserved on Frequency to form display handle as `base_handle`.`suffix`.
```